modules/manual-configuration-image-registry-ocp.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="manual-configuration-image-registry-ocp_{context}"]
+= Manually configuring {ocp} registry
+
+You can integrate {product-title} with {ocp} built-in container image registry.
+
+.Prerequisites
+* You need a *Username* and a *Password* for authentication with the {ocp} registry.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Image Integrations* section, select *Generic Docker Registry*.
+The *Configure image integration* modal box opens.
+. Click *New Integration*.
+. Enter the required details for:
```

Comment:
These are a bit nitty...
* If the UI provides a visual cue for the fields that are required, then I think you can safely remove "required."
* Consider revising in one of the following ways:
** Enter the details for the <registry | scanner>: (In this file, it's a registry. In other files, they are scanners, IIRC.)  This is my pref.
** Enter the details for the following fields:  (Ending with "for:" was too eye-catching for _me_.)